tests/regression/facebook/test_firefox_facebook_ail_scroll_home_1_txt.py

```diff
@@ -5,6 +5,9 @@ class TestSikuli(PerfBaseTest):
 
     def setUp(self):
         super(TestSikuli, self).setUp()
+        self.set_configs(self.config_name.INDEX,
+                         self.extract_platform_dep_settings({"win32": {"7": {"compare-threshold": 0.06},
+                                                                       '10': {'compare-threshold': 0.08}}}))
```

Comment:
threshold too large and cannot find correct result from local test on windows10 machine 6